front/app/containers/Admin/reporting/components/ReportBuilder/Editor/index.tsx

```diff
@@ -16,7 +15,7 @@ import RenderNode from './RenderNode';
 type EditorProps = {
   children: React.ReactNode;
   isPreview: boolean;
-  onNodesChange?: (query: QueryCallbacksFor<typeof QueryMethods>) => void;
+  onNodesChange?: React.ComponentProps<typeof CraftEditor>['onNodesChange'];
```

Comment:
`@craftjs/utils` was used only in this single file and not really needed. Better to delete it than have an upgrade break all of our tests again 😅